fluss-server/src/test/java/com/alibaba/fluss/server/testutils/FlussClusterExtension.java

```diff
@@ -142,6 +144,11 @@ public void afterEach(ExtensionContext extensionContext) {
             for (String database : databases) {
                 if (!database.equals(defaultDb)) {
                     metaDataManager.dropDatabase(database, true, true);
+                    // delete the data dirs
+                    for (int serverId : tabletServers.keySet()) {
```

Comment:
need to delete data dirs, otherwise, restart a server will meet problem since the metadata is dropped.